src/Mobs/Villager.cpp

```diff
@@ -203,7 +223,17 @@ bool cVillager::IsBlockFarmable(BLOCKTYPE a_BlockType)
 		{
 			return true;
 		}
+		default: return false;
 	}
-	return false;
 }
 
+
+
+
+
+cVillager::eVillagerType cVillager::GetRandomProfession()
+{
+	int Profession = GetRandomProvider().RandInt(cVillager::eVillagerType::vtMax - 1);
```

Comment:
This no longer encodes the increased farmer probability, is that intended?

Reply:
Yes